test/unit/mdc-top-app-bar/short.foundation.test.js

```diff
@@ -52,7 +52,7 @@ test('short top app bar: scrollHandler does not call getViewportScrollY method '
   const {foundation, mockAdapter} = setupTest();
   td.when(mockAdapter.hasClass(MDCTopAppBarFoundation.cssClasses.SHORT_COLLAPSED_CLASS)).thenReturn(true);
   foundation.init();
-  td.verify(mockAdapter.getViewportScrollY(), {times: 0});
+  td.verify(mockAdapter.getViewportScrollY(), {times: 1});
```

Comment:
Update test description accordingly.

Reply:
Done!
And I have one question...
If I want to request PR confirmation, I need to annotate some maintainer?
I'm used to MDCReact so I just leave PR until 1 week.